lib/player_creator.rb

```diff
@@ -0,0 +1,11 @@
+require 'pry'
+class PlayerCreator
```

Comment:
So, right now you pass an instance of `PlayerCreator` into your `Game` class. IMO, this is a bit backwards. I might have `PlayerCreator` return a new instance of `Game` rather than the other way around. This code is probably fine for this particular story, but you might run into architectural issues when you implement computer players

Reply:
No need to act on this comment if you don't agree with me, BTW. Just want to push back on this design decision.